README.md

```diff
@@ -18,7 +18,16 @@ Next, grab this React Native module with `npm install react-native-smooch`
 
 ## iOS
 
- * Navigate to your React Native project's `ios` directory and follow [these steps](http://docs.smooch.io/ios/#adding-smooch-to-your-app) for adding the Smooch binary distribution to your project with CocoaPods.
+ * Add react-native-smooch to your `Podfile`:
```

Comment:
It would be nice to keep the link to http://docs.smooch.io/ios/#adding-smooch-to-your-app there, so that a non-cocoapods install option remains discoverable.